killbill/payment_method.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2018 Quentral S.r.l.
```

Comment:
Sorry if I wasn't clear: we require a full copyright assignment for contributions (i.e. single copyright line to The Billing Project, LLC). This is to avoid potential problems in the future (e.g. future claims).

Reply:
I've seen in the other files that the copy is shared also with other companies (Groupon ecc..) that are involved in the library creation. Considering that I've updated the library during the work time of Quentral s.r.l., I need to check with my CEO if is it fine to let the copy only to The Billing Project, LLC.

Reply:
The companies currently listed are the major sponsors of the project.

We insist on a full copyright assignment, especially on such small changes, because our project doesn't have the resources to track down individual authors for a license to redistribute the code.

I know this is a bit annoying but it is to make our life, as maintainers, easier, and to protect the project moving forward.

Reply:
I'm not sure this contribute will become a "small change" (I have to add some other classes and methods in the near future), but I talked yesterday with my CEO and it is fine to remove our copy line.
In the last commit you find the line removed.